### PR TITLE
feat: usage statistics

### PR DESCRIPTION
**Pull request by robertsLando**
Needs https://github.com/zwave-js/node-zwave-js/pull/2207

cc @AlCalzone

### CODE REVIEW

File: lib/ZwaveClient.js [hunk 1]
```diff
@@ -1753,7 +1754,15 @@ ZwaveClient.prototype.connect = async function () {
...
+        },
+        metrics: this.cfg.disableMetrics
+          ? null
+          : {
+              applicationName:
+                pkgjson.name +
+                (this.cfg.serverEnabled ? '/ zwave-js-server' : ''),
+              applicationVersion: pkgjson.version
+            }
       },
```
Comment (AlCalzone) [inline lib/ZwaveClient.js:1766]: I've switched to two methods to enable/disable sending usage statistics.

File: src/components/Settings.vue [hunk 2]
```diff
@@ -161,6 +161,14 @@
                           @click:append-outer="randomKey"
                         ></v-text-field>
                       </v-col>
+                      <v-col cols="12" sm="6">
+                        <v-switch
+                          hint="Metrics allows us to gain insight how `zwave-js` is used, which manufacturers and devices are most prevalent and where to best focus our efforts in order to improve `zwave-js` the most. We do not store any personal information. Details can be found under https://zwave-js.github.io/node-zwave-js/#/getting-started/telemetry.md#usage-statistics"
```
Comment (AlCalzone) [inline src/components/Settings.vue:166]: Maybe we should do one iteration with @blhoward2 on how to formulate this best. Also affects zwave-js docs.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
metrics -> usage statistics :)

File: lib/ZwaveClient.js (hunk 1 above)
Comment (AlCalzone) [reply lib/ZwaveClient.js:1766]: https://github.com/zwave-js/node-zwave-js/blob/metrics/docs/api/driver.md#enablestatistics

File: src/components/Settings.vue (hunk 2 above)
Comment (robertsLando) [reply src/components/Settings.vue:166]: 👍🏼

File: lib/ZwaveClient.js (hunk 1 above)
Comment (robertsLando) [reply lib/ZwaveClient.js]: Fixed

**Pull request merged**